nautobot_circuit_maintenance/api/urls.py

```diff
@@ -8,7 +8,7 @@
 router.register("maintenance", views.MaintenanceTaskView)
 router.register("note", views.MaintenanceNoteTaskView)
 router.register("circuitimpact", views.MaintenanceCircuitImpactTaskView)
-router.register("notificationsource", views.NotificationSourceTaskView)
+router.register("source", views.NotificationSourceTaskView)
 router.register("parsednotification", views.ParsedNotificationTaskView)
 router.register("rawnotification", views.RawNotificationTaskView)
```

Comment:
If we're making breaking changes anyway, *all* of these are wrong - should be `maintenances`, `notes`, `circuit-impacts`, `sources`, `parsed-notifications`, `raw-notifications`, etc.

Reply:
nice to know, where is this convention defined?